app/experimenter/experiments/tests/test_tasks.py

```diff
@@ -369,29 +361,20 @@ def test_experiment_with_re_enabled_enrollment(self):
         )
 
     def test_experiment_with_paused_staying_the_same(self):
-        ExperimentFactory.create_with_status(
+        experiment = ExperimentFactory.create_with_status(
             target_status=Experiment.STATUS_LIVE,
             normandy_id=1234,
             is_paused=False,
         )
 
-        self.mock_normandy_requests_get.return_value = (
-            self.buildMockSucessWithNoPauseEnrollment()
-        )
-        tasks.update_experiment_info()
```

Comment:
Ah I realized something after we talked the other day, I called this 'cheating' but here's a concrete example:

If you deleted `set_is_paused_value(...)` from `update_experiment_info`, then this test would still pass but it when the task actually ran it would be broken, so it's generally good to test from the entry point that will get used in when the thing actually runs.